src/%IPM/Storage/ResourceReference.cls

```diff
@@ -120,7 +120,8 @@ ClassMethod GetChildren(pResourceName As %String, pModuleName As %String, pCheck
 					Set tResourceCache(tCacheResult.%Get("UniqueName")) = ""
 				}
 			}
-			
+			// include hidden files
+			Set ^%SYS("Studio","ShowHidden") = 1
```

Comment:
If this is a global setting, we should not be setting it and leaving it as is. Instead, should record the previous value, set it to 1 before the query and reset it to the previous value after the query